models/demos/t3000/mixtral8x7b/demo/demo_with_prefill.py

```diff
@@ -312,7 +312,7 @@ def test_mixtral8x7b_demo(t3k_device_mesh, use_program_cache, input_prompts, ins
 
 
 # CI only runs general-weights demo
-@pytest.mark.skipif(os.getenv("CI") == "False", reason="CI-only test")
+@pytest.mark.skipif(not os.getenv("CI") == "true", reason="CI-only test")
```

Comment:
Please use this https://github.com/tenstorrent/tt-metal/pull/10371/files

Reply:
This is still not using the fixture?

Reply:
This PR only touches Mistral.

This change was a small bug I noticed and did a quick patch inside this PR. 

We will open a new PR with the changes on Mixtral side after this one.